cmd/libs/go2idl/client-gen/generators/generator_for_group.go

```diff
@@ -167,7 +176,7 @@ func New(c *$.RESTClient|raw$) *$.Group$Client {
 	return &$.Group$Client{c}
 }
 `
-var setClientDefaultsTemplate = `
+var legacySetClientDefaultsTemplate = `
```

Comment:
I don't think legacy is descriptive here-- maybe setClientInternalVersionDefaultsTemplate?